crates/apub/src/api/list_comments.rs

```diff
@@ -37,24 +35,31 @@ impl PerformApub for GetComments {
     let local_site = LocalSite::read(context.pool()).await?;
     check_private_instance(&local_user_view, &local_site)?;
 
-    let community_id = data.community_id;
-    let listing_type = listing_type_with_site_default(data.type_, &local_site)?;
-
-    let community_actor_id = if let Some(name) = &data.community_name {
+    let community_id = if let Some(name) = &data.community_name {
       resolve_actor_identifier::<ApubCommunity, Community>(name, context, &None, true)
         .await
         .ok()
-        .map(|c| c.actor_id.clone())
+        .map(|c| c.id)
     } else {
-      None
+      data.community_id
     };
-    let sort = data.sort;
+    let sort = data.sort.unwrap_or_default();
     let max_depth = data.max_depth;
     let saved_only = data.saved_only;
     let page = data.page;
     let limit = data.limit;
     let parent_id = data.parent_id;
 
+    // On frontpage use listing type from param or admin configured default
```

Comment:
This is copy pasted below, might as well use a helper function.

Reply:
True, I didnt do that before because with community_id and community_actor_id, there would have been too many parameters and it would be awkward. Added helper now.